modules/apps/27-interchain-accounts/controller/keeper/account.go

```diff
@@ -65,16 +64,7 @@ func (k Keeper) registerInterchainAccount(ctx context.Context, connectionID, por
 		return "", errorsmod.Wrapf(icatypes.ErrActiveChannelAlreadySet, "existing active channel %s for portID %s on connection %s", activeChannelID, portID, connectionID)
 	}
 
-	switch {
-	case k.portKeeper.IsBound(ctx, portID) && !k.hasCapability(ctx, portID):
-		return "", errorsmod.Wrapf(icatypes.ErrPortAlreadyBound, "another module has claimed capability for and bound port with portID: %s", portID)
-	case !k.portKeeper.IsBound(ctx, portID):
```

Comment:
This case is - if no capability exists (port bound) for port id in core ibc's scoped keeper, then;
- Set port: this was previously used for genesis exports such that we can recreate ports on a genesis restart, unclear if we still actually need this for ica as ports are assumed to be in the format `icacontroller-`. Unlike transfer where we set the port because it could be modified by devs to e.g. "ibc-transfer" (unclear if there is assumptions in other places that it must just be "transfer").
- Create a new capability via `BindPort` using ibc core's scoped keeper. This creates a capability and adds "ibc" as an owner.
- Claim the capability using the ica scoped keeper. This adds ica as the second owner

In scenarios where we open a new channel(after close) for an existing port id, then neither of these cases are hit and skip the entire switch statement.